docs/cloud/index.md

```diff
@@ -35,30 +35,33 @@ Prefect Cloud includes all the features in the open-source Prefect server plus t
     - [Workspaces](/cloud/workspaces/) &mdash; isolated environments to organize your flows, deployments, and flow runs.
     - [Automations](/cloud/automations/) &mdash; configure triggers, actions, and notifications in response to real-time monitoring events.
     - [Email notifications](/cloud/automations/) &mdash; send email alerts from Prefect's server based on automation triggers.
-    - [Organizations](/cloud/organizations/) &mdash; user and workspace management features that enable collaboration for larger teams.
     - [Service accounts](/cloud/users/service-accounts/) &mdash; configure API access for running workers or executing flow runs on remote infrastructure.
-    - [Custom role-based access controls (RBAC)](/cloud/users/roles/) &mdash; assign users granular permissions to perform certain activities within an organization or a workspace.
+    - [Custom role-based access controls (RBAC)](/cloud/users/roles/) &mdash; assign users granular permissions to perform certain activities within an account or a workspace.
     - [Single Sign-on (SSO)](/cloud/users/sso/) &mdash; authentication using your identity provider.
     - [Audit Log](/cloud/users/audit-log/) &mdash; a record of user activities to monitor security and compliance.
-    - Collaborators &mdash; invite others to work in your [workspace](/cloud/workspaces/#workspace-collaborators) or [organization](/cloud/organizations/#organization-members).
+    - Collaboration &mdash; invite other people to your account.
     - Error summaries  &mdash; (enabled by Marvin AI) distill the error logs of `Failed` and `Crashed` flow runs into actionable information.
-    - [Push work pool](/guides/deployment/push-work-pools/) &mdash; run flows on your serverless infrastructure without running a worker.
+    - [Push work pools](/guides/deployment/push-work-pools/) &mdash; run flows on your serverless infrastructure without running a worker.
 
 ## User accounts
 
-When you sign up for Prefect Cloud, a personal account is automatically provisioned for you.
-A personal account gives you access to profile settings where you can view and administer your:
+When you sign up for Prefect Cloud, an account is automatically provisioned for you.
+An account gives you access to profile settings where you can view and administer your:
```

Comment:
I think we want to separate the idea of the Profile (i.e. User) and account here. 

Maybe something like:

When you sign up for Prefect Cloud, and account and a profile are automatically created for you.

Your profile is the place where you'll manage settings related to your user, including:
- Profile, including profile handle and image
- API keys
- Preferences, including timezone and color mode

As an account Admin, you will also have access to settings from the account settings page, such as:
- Billing
- Feature previews, if available
- Roles
- Account Membership
etc